README.md

```diff
@@ -96,6 +97,36 @@ WelcomeEmail.new("Kate", "kate@example.com").deliver
 WelcomeEmail.new("Kate", "kate@example.com").deliver_later
 ```
 
+### Delay email delivery
```

Comment:
Since we are publishing api docs now, could these docs be moved to the classes since the get added there? The README could be updated to say go there for docs

Reply:
Yeah, probably eventually. That's going to be a pretty big undertaking adding in all those docs, so we can save that for another day.